### PR TITLE
docs(SCT-169): add documentation for connecting to a database

### PR DESCRIPTION
**Pull request by chubberlisk**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/SCT-169

### *What is the problem we're trying to solve*

We don't have documentation on how to connect to a database within the repo.

### *What changes have we introduced*

This PR adds documentation for how to connect to the PostgreSQL and MongoDB in an environment and updates the Updating the database schema doc to use this.

### CODE REVIEW

File: docs/connecting-to-a-database.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Connecting to a database
+
+> ⚠️ **Warning**: Particularly if you're connecting to a production database,
+> proceed with caution and look to pair with someone else.
```
Comment (Renny4Real) [inline docs/connecting-to-a-database.md:4]: Love this!

File: docs/connecting-to-a-database.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+| Environment | AWS account       | Systems Manager key | EC2 jumpbox                      |
+|-------------|-------------------|---------------------|----------------------------------|
+| Staging     | StagingAPIs       | staging             | RDS jump box-Platform APIs (new) |
+| Production  | Mosaic-Production | mosaic-prod         | bastion-rds-jump-box-prod        |
+
+## Connecting to the PostgreSQL (RDS PostgreSQL) database
+
+1. Within relevant AWS account, go to **Systems Manager** → **Parameter Store**
+2. Search for **/social-care-case-viewer-api/\<environment-key\>/postgres**
```
Comment (Renny4Real) [inline docs/connecting-to-a-database.md:25]: Is the Systems Manager key the name we expect to see when looking for the parameters?

Perhaps, this should be:
```suggestion
2. Search for **/social-care-case-viewer-api/\<systems-manager-key\>/postgres**
```

File: docs/connecting-to-a-database.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+3. In a new tab, go to **Session Manager** under **Node Management** in **Systems Manager**
+4. Click on **Start session**
+5. Select the appropriate EC2 jumpbox and click on **Start session**
+6. Using the parameters from Parameter Store in your other tab, connect to the PostgreSQL database by running:
+
+```
+$ psql --host=<hostname> --port=<port> --username=<username> --dbname=social_care
+```
```
Comment (Renny4Real) [inline docs/connecting-to-a-database.md:33]: We could mention here that you would be prompted for a password and where to get it depending on the environment—e.g. secrets manager for the production, parameter store for the staging account, etc.

File: docs/connecting-to-a-database.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+$ wget https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem
+```
+
+6. Using the parameter from Parameter Store in your other tab, connect to the MongoDB by running the following command and extracting parts from the connection string - `mongodb://<USERNAME>:<PASSWORD>@<HOSTNAME>:<PORT>`
+
+```
+mongo --ssl --host <HOSTNAME>:<PORT> --sslCAFile rds-ca-2019-root.pem --username <USERNAME> --password
+```
+
+7. Paste the password and hit enter
```
Comment (Renny4Real) [inline docs/connecting-to-a-database.md:80]:
```suggestion
7. Paste the password after the prompt and hit enter
```

Wondering if someone might read this as paste it after `--password` ?

**Review by Renny4Real — COMMENTED** [review]
Looks good 🎉

File: docs/connecting-to-a-database.md (hunk 2 above)
Comment (chubberlisk) [reply docs/connecting-to-a-database.md:25]: Good spot, thanks!

File: docs/connecting-to-a-database.md (hunk 1 above)
Comment (chubberlisk) [reply docs/connecting-to-a-database.md:4]: Stolen idea from a doc Chris made somewhere for the frontend. 😜

File: docs/connecting-to-a-database.md (hunk 3 above)
Comment (chubberlisk) [reply docs/connecting-to-a-database.md:33]: I update the doc with hopefully better clarity, let me know what you think of https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/323/commits/16ce6d4fb8fc60695343293d1fa46829e3b77eff.

File: docs/connecting-to-a-database.md (hunk 4 above)
Comment (chubberlisk) [reply docs/connecting-to-a-database.md:86]: I updated the doc with hopefully better clarity, let me know what you think of https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/323/commits/16ce6d4fb8fc60695343293d1fa46829e3b77eff.

**Review by Renny4Real — APPROVED** [review]
🎉

**Pull request merged**